components/Badge/MarketBadge.tsx

```diff
@@ -43,15 +43,19 @@ export const MarketBadge: FC<MarketBadgeProps> = ({
 	const timerSetting = isOpen === null ? null : isOpen ? nextTransition : nextOpen;
 	const isMarketTransitioning = useIsMarketTransitioning(timerSetting ?? null);
 
-	if (isMarketTransitioning && isOpen !== null) {
-		return <TransitionBadge isOpen={isOpen} />;
+	if (typeof isFuturesMarketClosed !== 'boolean') {
+		return null;
```

Comment:
dont even show a badge unless we have a [legit value for if the market is closed or not.]()